src/axolotl/utils/samplers/multipack.py

```diff
@@ -136,6 +187,11 @@ def __init__(
         # the minimum packed dataset length across all ranks determined by a gather/broadcast
         self.len_across_ranks = None
 
+        if self.sequential and not isinstance(sampler, SequentialSampler):
+            LOG.warn(
+                "using sequential sample packing with non-sequential sampler, did you want to also enable curriculum_sampling?"
+            )
+
```

Comment:
Similar to the above comment, could we move this validation to the schema `src/axolotl/utils/schemas/config.py` and not here?

Reply:
You can change the check to be: if sample_pack_sequential and not curriculum_sampling instead